imports/plugins/core/ui/client/containers/buttonSelectContainer.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component, PropTypes } from "react";
+import classnames from "classnames";
+import { composeWithTracker, merge } from "/lib/api/compose";
+import Button from "../components/button/button.jsx";
+import { ButtonSelect } from "../components/button";
+
+class ButtonSelectContainer extends Component {
```

Comment:
So basically, most of this component looks presentational, and should be moved into a `Presentational` component.

Everything related to composeWithTracker, etc, should stay.